library/src/scala/quoted/Quotes.scala

```diff
@@ -4364,7 +4360,7 @@ trait Quotes { self: runtime.QuoteUnpickler & runtime.QuoteMatching =>
       *   -  ...
       *   -  Nth element is `FunctionN`
       */
-      @deprecated("Use overload of `FunctionClass` with 1 or 2 arguments","3.4")
+      @deprecated("Use overload of `FunctionClass` with 1 or 2 arguments", "3.4")
```

Comment:
unrelated to this PR but this message seems flaky if we were to add another overload with 1 or 2 arguments